kivy/core/window/__init__.py

```diff
@@ -528,6 +528,21 @@ def _get_system_size(self):
     '''Real size of the window ignoring rotation.
     '''
 
+    def _get_effective_size(self):
+        w, h = self.system_size
+        if platform == 'ios' or self._density != 1:
+            w, h = self.size
+
+        return w,h
+
+    effective_size = AliasProperty(_get_effective_size, None)
+    '''On density=1 and non-ios displays, return system_size, else
+    return scaled / rotated size.
+
```

Comment:
My reasons for doing it this way was indeed to keep it DRY (having two different places checking the same logic is a bit fragile), and to follow the lead of the other properties.

Would you prefer that we rather just remove the property, and then rename the function to "get_effective_size()" (without underscore) and use that?